Remark on posting solutions in discourse after solving a kata (#341)
codewars/docs: content/training/troubleshooting.mdx

- Use [code formatting](/references/markdown/#code-block) when posting blocks of code.
- Use the **spoiler flag** when your post contains parts of a solution (even a non-working one), or if it mentions anything that might reveal details of the solution.

> - Do not post your solution in discourse after you complete a kata. It's not really necessary, and if anyone is interested, they can always see it on "Solutions" tab on a kata page.

Reviewer: ```suggestion
- Do not post your solution in the discourse after you complete a kata. It's not necessary, and if anyone is interested, they can look it up under the "Solutions" tab of the kata.
```
